src/TestUtils/EventuallyConsistentTestTrait.php

```diff
@@ -17,7 +17,12 @@
 
 namespace Google\Cloud\TestUtils;
 
-use PHPUnit_Framework_ExpectationFailedException;
+if (!class_exists('\PHPUnit\Framework\ExpectationFailedException', true)) {
```

Comment:
Shouldn't this be done in the `bootstrap.php` file? Putting it at the top of a class file is pretty dirty.

Reply:
I wanted this file to be usable for both phpunit4 users and phpunit6 users, without adding anything in the bootstrap file.

Reply:
A different way to do this then would be to catch all exceptions and check the classname, or alternately add a method inside the trait such as `aliasExpectationFailedException` and call it before the try/catch.